### DISCUSSION TITLE
Giphy integration

### DISCUSSION THREAD

**Discussion by BartoszJarocki** [ParabolInc/parabol#6359] [Ideas]
Context: https://parabol.slack.com/archives/C836NA350/p1649250771147569

The question is, should we integrate Giphy into Parabol’s discussion threads?
I personally think this is a good idea and would love to see it in Parabol. 

Any thoughts?

**Comment by mattkrick** [comment]
Thoughts:
- Let's use tenor instead of giphy. tenor usually has better results & giphy is owned by facebook 
- Let's enable it by adding a tenor extension to tiptap. maybe 1 exists already! then we'll have an incentive to refactor draftjs components to tiptap
